src/TitleBindPieces/TitleBindPiecesContainer/TitleBindPiecesContainer.js

```diff
@@ -50,7 +50,7 @@ export const TitleBindPiecesContainer = () => {
   } = useTitleHydratedPieces({
     titleId,
     receivingStatus: PIECE_STATUS.received,
-    searchQuery: 'isBound==false',
+    searchQuery: 'isBound==false and format==Physical',
```

Comment:
There is a constant with this value https://github.com/folio-org/stripes-acq-components/blob/e6ed48a1fcdcc1658c2795d23760fd902bc4e95f/lib/constants/piece.js#L13

Reply:
updated